### PR TITLE
Populate the 'visibility' field on Item edit form.

### PR DESCRIPTION
**Pull request by cwant**
App was not filling in visibility correctly when editing an item, now it does. Although the current form doesn't seem to have an 'embargo' visibility -- oversight or feature?

### CODE REVIEW

File: app/views/items/_form.html.erb [hunk 1]
```diff
@@ -4,9 +4,10 @@
     <%= f.input prop %>
   <% end %>
   <%= f.input :visibility,
-  as: :unread_select,
-  collection: ['public', 'private', 'authenticated'],
-  input_html: {name: 'item[visibility]'}
+    as: :unread_select,
+    collection: ['public', 'private', 'authenticated'],
```
Comment (murny) [inline app/views/items/_form.html.erb]: Could be a separate PR, but we should be using the `JupiterCore::VISIBILITY_PUBLIC` etc constants (or if this array list is going to be popular create a array for this in JupiterCore::VISIBILITIES or something) for this instead of hardcoding strings

Otherwise looks good

Comment (cwant) [reply app/views/items/_form.html.erb]: I've done this as a bit of a hierarchical thing (`Item` has one extra visibility that `JupiterCore` doesn't).
